app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -18,49 +18,81 @@ class GetUspsProofingResultsJob < ApplicationJob
 
   discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
 
+  def enrollment_attributes(enrollment, enrollment_completed)
```

Comment:
Would it make sense for this method to be `private` ? Also mentioning since stylistically I'd prefer `perform` to be the topmost method, but I also think it's the only "public" method.

Reply:
Derp, yes. Thanks

Reply:
I wonder if it would also be good to name this method to clarify what kind of attributes it's generating, since they're specific to analytics, e.g. `enrollment_analytics_attributes`.

Reply:
Done in 2325a35385b0347cfc944a0a760b7bdba879df1d